site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Integrate a custom Job"
+date: 2022-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+This page shows how to integrate a custom Job with Kueue. 
+
+## What we need to build
+
+1. Configration
+2. Controller
+3. Webhook
+
+## Configure your custom Job
+
+Add your framework name to `.integrations.frameworks` in `config/components/manager/controller_manager_config.yaml`
+
+## Build controller
+
+You can create the controller in `./pkg/controller/jobs/$YOUR_CUSTOM_JOB`.
+
+1. Register a new framework
+2. Add RBAC Authorization
+    - Add RBAC Authorization for priorityclasses, events, workloads, resourceflavors and any resources you want.
+3. Implement interfaces
```

Comment:
I'm wondering if repeating the interface here is worth it. It adds maintenance cost to keep in sync. Maybe just a link to the interface would do.